openedx/core/djangoapps/profile_images/views.py

```diff
@@ -2,6 +2,8 @@
 This module implements the upload and remove endpoints of the profile image api.
 """
 from contextlib import closing
+import hashlib
+import time
```

Comment:
It merged into jsa/profile-image-api which then should have been merged into learner-profiles: https://github.com/edx/edx-platform/pull/7458